plugin/pkg/scheduler/algorithm/priorities/priorities_test.go

```diff
@@ -929,3 +930,142 @@ func TestPrioritiesRegistered(t *testing.T) {
 		}
 	}
 }
+
+func TestNodePreferAvoidPriority(t *testing.T) {
```

Comment:
Please also also test the case where pod (and annotation) has a ReplicaSet instead of a RepilcationController, once you add the replica set lister.